db/load_history.sql

```diff
@@ -0,0 +1,21 @@
+-- download 2020.zip from 
+-- https://elections.sos.ga.gov/Elections/voterhistory.do
+-- unzip it and then run this script.
```

Comment:
i suppose we might want to add some more instructions to the readme and maybe something in the Makefile to give info about how to do this...welcome suggestions about what to put in Makefile